src/server/ua_server.c

```diff
@@ -128,8 +128,8 @@ void UA_Server_delete(UA_Server *server) {
 
 #ifdef UA_ENABLE_SUBSCRIPTIONS
     UA_MonitoredItem *mon, *mon_tmp;
-    LIST_FOREACH_SAFE(mon, &server->localMonitoredItems, listEntry, mon_tmp) {
```

Comment:
Revert renaming to listEntry_store.
This adds only noise to the PR.

This can be added to a "cosmetic changes" PR.